demo/android/OrcaDemo/orca-demo-app/src/main/java/ai/picovoice/orcademo/MainActivity.java

```diff
@@ -53,6 +62,8 @@ public class MainActivity extends AppCompatActivity {
 
     private final Handler mainHandler = new Handler(Looper.getMainLooper());
     private final ExecutorService executor = Executors.newSingleThreadExecutor();
+    private final ExecutorService executor1 = Executors.newSingleThreadExecutor();
```

Comment:
would be good to give these more descriptive names like `executor_synthesis`, `executor_audio`, or something